From 7ef074673491271596e500db8098ff0c04b370aa Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Thu, 8 Oct 2020 11:00:05 -0400
Subject: [PATCH] docs: Don't build man pages on Windows.

There's no man utility to read them, and bzip2 would be an extra
non-standard dependency.
---
 docs/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/docs/CMakeLists.txt b/docs/CMakeLists.txt
index 6aba5e4..d19169d 100644
--- a/docs/CMakeLists.txt
+++ b/docs/CMakeLists.txt
@@ -39,6 +39,8 @@ endif(ENABLE_DOXYGEN)
 ########################################################################
 # Man pages
 ########################################################################
+if(NOT WIN32)
+
 FIND_PROGRAM(BZIP
              NAMES bzip2
              PATHS /bin
@@ -70,3 +72,5 @@ foreach(manpage ${manpages})
        DESTINATION ${MAN_INSTALL_DIR}/man1
     )
 endforeach(manpage)
+
+endif(NOT WIN32)
-- 
2.20.1

